examples/ota-requestor-app/linux/README.md

```diff
@@ -212,7 +215,7 @@ scripts/examples/gn_build_example.sh examples/ota-requestor-app/linux/ out chip_
 **Run the OTA Requestor application**
 
 ```
-out/chip-ota-requestor-app --discriminator 18 --secured-device-port 5560 --KVS /tmp/chip_kvs_requestor --delayQuery 30
+out/chip-ota-requestor-app --discriminator 18 --secured-device-port 5560 --KVS /tmp/chip_kvs_requestor --delayQuery 30 --otaDownloadPath /tmp/test.bin
```

Comment:
Should we add instructions here that ota_image_tool.py can be used to generate/verify valid OTA image header before using here?

Reply:
There's already mention of that here: https://github.com/project-chip/connectedhomeip/tree/master/examples/ota-requestor-app/linux#software-image-header

The assumption is that by the time you get to seeing the examples, you've got a valid test file